test/service/test_response.py

```diff
@@ -1895,8 +1898,20 @@ def test_empty_response(self):
         response.raise_for_status()
         response = response.json()
         self.assertEqual([], response['hits'])
-        self.assertEqual({None}, set(response['pagination'].values()))
-        self.assertEqual({}, response['termFacets'])
+        self.assertEqual(0, response['pagination']['count'])
+        self.assertEqual(0, response['pagination']['total'])
+        for _, value in response['termFacets'].items():
```

Comment:
If you don't need the keys, you shouldn't use `.items()`

Reply:
If esponse['termFacets'] is empty, no assertions will be made. I don't think that's what you want.